packages/site/src/layout/NavMenu.module.css

```
@@ -0,0 +1,98 @@
+.navMenuContainer {
+    --hoverOffset: var(--space-smallest);
+    --negativeHoverOffset: calc(-1 * var(--hoverOffset));
+    --subItemInset: calc(var(--space-small) + var(--space-smaller));
+    --navItemHeight: 40px;
+    display: flex;
+    flex-direction: column;
+    padding: 36px 0 0 0;
+    height: 100dvh;
+    background-color: var(--color-surface--background);
+    width: 200px;
+    box-sizing: border-box;
+}
+
+.navMenuContainer > * {
```

Could change this declaration to 

```suggestion
.navMenuHeader,
.navMenu {
```

if preferred